index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Week1</title>
+      <link rel="stylesheet" href="styles/style.css">
+   </head>
+   <body>
+      <header>
+         <div>
+            <div>
+               <a href=""> <img src="images/swiggylogo.png" alt = "Swiggy Logo" class = "swiggy-logo"> </a>
+               <a href="">Set location</a> 
+               <a href="">Click to Choose Location</a>
+            </div>
+            <ul>
+               <li><a href="">Help</a></li>
+               <li><a href="">Sign In</a></li>
+               <li><a href="">Cart</a></li>
+            </ul>
+         </div>
+      </header>
+      <section class = "nav-bar">
+         <nav>
+            <ul>
+               <li><a href="">Home</a> <span>/</span></li>
+               <li><a href="">Banglore</a> <span>/</span></li>
+               <li><a href="">Ashok Nagar</a> <span>/</span></li>
+               <li><a href="">Meghana Foods</a></li>
+            </ul>
+         </nav>
+      </section>
+      <section class="banner">
+         <img src="images/box1_image.jpeg" alt = "Food Image">
+         <h1>Punjabi Momos</h1>
+      </section>
+      <main class="row">
+         <div class="categories col-3">
+            <ul>
+               <li class = "category-recommended">
+                  Recommended
+               </li>
+               <li>
+                  Chandano chowk ki birani
+               </li>
+               <li>
+                  Naughty Momos
+               </li>
+               <li>
+                  Naughty Momos Roll
+               </li>
+            </ul>
+         </div>
+         <div class="menu-items col-5">
+            <div class="category-info">
+               <h3>Recommended</h3>
+               <p>24 Items</p>
+            </div>
+            <div class="item-description">
+               <img src="images/veg_mark.png" alt="Veg Mark Logo">
+               <p class="dish-name">Kadhai Paneer Biryani</p>
+               <p> &#8377 249</p>
+            </div>
+            <div class="item-description">
+               <img src="images/veg_mark.png" alt="Veg Mark Logo">
+               <p class="dish-name">Masala Chaap Biryani</p>
+               <p> &#8377 249</p>
+            </div>
+         </div>
+         <div class="cart col-2">
+            <div>
+               <h3>Cart</h3>
+               <p>1 Items</p>
+               <br>
+               <p class="dish-name">Kadhai Paneer Biryani</p>
+               <span>
+                  <p>Subtotal</p>
+                  <p> &#8377 249</p>
+               </span>
+               <p id="charges-description">Extra charges may apply</p>
```

Comment:
This should be a class and not an id.

An id is only used when you want to uniquely identify some element for some reason.

Same goes for the checkout button as well